code.rb

```diff
@@ -1,13 +1,13 @@
 module Enumerable
   def my_each
     for i in 0...self.length
-      yield (self[i])
+      yield(self[i])
```

Comment:
:100:

Reply:
I made some of the changes suggested but there are other that I don't know how to resolve,
For example in the methods my_each and my_each_with_the_index the stickler says that I should replace for by each but if what I'm trying to do it, is not use each.
And about the erros of self, I dont how to resolve it too

Reply:
You can use other loop methods, have you tried `while` loop? :thinking:

Reply:
Thanks a lot! Changes made!